zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -106,7 +106,7 @@
               <li ng-if="ticket.principal && ticket.principal !== 'anonymous'" role="separator" style="margin: 5px 0;" class="divider"></li>
               <li ng-if="ticket.principal && ticket.principal !== 'anonymous'"><a ng-click="navbar.logout()">Logout</a></li>
               <li role="separator" style="margin: 5px 0;" class="divider"></li>
-              <li><a href="/next">Try the new Zeppelin</a></li>
+              <li><a href="/">Try the new Zeppelin</a></li>
```

Comment:
Are you suggesting that we should change the "Try the new Zeppelin" text?
I initially decided to keep it the same even after switching the default UI, thinking it would be natural for old UI users to see the same wording guiding them to the new UI.
Do you have any alternative wordking in mind?

Reply:
might be "Classic UI" vs. "Default UI"

Reply:
As you mentioned, using the term "Default UI" seems natural.
How about this phrase: "Switch to the default UI"

Reply:
@tbonelee I replied here https://github.com/apache/zeppelin/pull/4802/files#r1731289824